app/models/effective/effective_datatable/rendering.rb

```diff
@@ -71,7 +71,8 @@ def arrayize(collection)
               edit_action: (opts[:partial_locals] || {})[:edit_action],
               destroy_action: (opts[:partial_locals] || {})[:destroy_action],
               unarchive_action: (opts[:partial_locals] || {})[:unarchive_action]
-            }
+            )
```

Comment:
It'd be nice if Hashes were indifferent by default. With the new string optimizations, maybe this is being considered??